README.md

```diff
@@ -33,6 +33,69 @@ Usage
 
 ### Minimal setup
 
+<details>
+    <summary><b>Azure Pipelines</b>
+        <img width="16" src="https://unpkg.com/simple-icons@latest/icons/apple.svg" />
+        <img width="16" src="https://unpkg.com/simple-icons@latest/icons/windows.svg" />
+        <img width="16" src="https://unpkg.com/simple-icons@latest/icons/linux.svg" />
+    </summary>
+
+- Using Azure pipelines, you can build all three platforms on the same service. Create a `azure-pipelines.yml` file in your repo.
+
+**azure-pipelines.yml**
+```yaml
+jobs:
+- job: linux
+  pool: {vmImage: 'Ubuntu-16.04'}
+  steps: 
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        pip install cibuildwheel==0.10.1
+        cibuildwheel --output-dir wheelhouse .
+    - task: PublishBuildArtifacts@1
+      inputs: {pathtoPublish: 'wheelhouse'}
+- job: macos
+  pool: {vmImage: 'macOS-10.13'}
+  steps: 
+    - task: UsePythonVersion@0
+    - bash: |
+        python -m pip install --upgrade pip
+        pip install cibuildwheel==0.10.1
+        cibuildwheel --output-dir wheelhouse .
+    - task: PublishBuildArtifacts@1
+      inputs: {pathtoPublish: 'wheelhouse'}
+- job: windows
+  pool: {vmImage: 'vs2017-win2016'}
+  steps: 
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '2.7', architecture: x86}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '2.7', architecture: x64}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.4', architecture: x86}}
+    - {task: UsePythonVersion@0, inputs: {versionSpec: '3.4', architecture: x64}}
```

Comment:
Should 3.4 be removed here since it won't be supported?

Reply:
Although I guess it's fine for pure-Python packages, so maybe fine to leave it.

Reply:
Python 3.4 has reached End-Of-Life, it can probably be removed.

Reply:
I've removed this from the Azure config. We can remove support for 3.4 from the entire cibuildwheel at some point, but as long as it's working I'm happy to leave it in for now.